src/librustdoc/lib.rs

```diff
@@ -34,15 +34,12 @@ extern crate rustc_interface;
 extern crate rustc_metadata;
 extern crate rustc_target;
 extern crate rustc_typeck;
-extern crate serialize;
 extern crate syntax;
```

Comment:
Hmm; unrelatedly to this PR, these `extern crate`s seem unfortunate; would be good to get rid of them.

Reply:
Aren't they necessary since they're being loaded from the sysroot?

Reply:
Possibly... I'm not familiar with how rustdoc operates. =P One could try to remove them and see what happens.

Reply:
Yeah, it loads all the rustc crates from the sysroot, not via `--extern` flags. Does rust 2018 allow you to implicitly load in sysroot crates?